### PR TITLE
fix: upgrade libraries to fix CVE issues

### LINKED ISSUE #239: Having CVE in the current version. can you please update to latest version of mssql and loopback-connector

### PR DESCRIPTION
**Pull request by mahesh-uma**
Fixes #239

Resolve CVE issues 

Signed-off-by: uma mahesh <mahesh456.t@gmail.com>

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -1,7 +1,7 @@
 {
   "extends": "loopback",
   "rules": {
-    "max-len": ["error", 125, 4, {
+    "max-len": ["error", 140, 4, {
```
Comment (dhmlau) [inline .eslintrc:4]: any special reason to increase the length?

**Review by dhmlau — COMMENTED** [review]
@mahesh-uma, thanks for the PR. I've ran the tests locally with your changes. LGTM. Just have one minor comment about the change in `.eslintrc`. 

Side note: we'd need to get the GitHub Actions running for CI.

File: .eslintrc (hunk 1 above)
Comment (mahesh-uma) [reply .eslintrc:4]: when I run npm test, giving error saying needed 140. so i changed  it. 
(how to run github actions running for CI, Did i miss some thing )

Comment (dhmlau) [reply .eslintrc:4]: Sorry about the confusion. The GitHub Actions comment is for future reference. :) 
Currently I'm verifying locally, but ideally we need to get GitHub Actions working for CI. But that's out of scope for this PR.

**Review by dhmlau — APPROVED** [review]

**Comment by dhmlau** [comment]
@mahesh-uma, thanks for the contribution. Your PR has landed! 🎉

**Pull request merged**